clusterman/autoscaler/autoscaler.py

```diff
@@ -83,7 +81,12 @@ def __init__(
 
         gauge_dimensions = {'cluster': cluster, 'pool': pool}
         monitoring_client = get_monitoring_client()
-        self.target_capacity_gauge = monitoring_client.create_gauge(TARGET_CAPACITY_GAUGE_NAME, gauge_dimensions)
+        self.target_capacity_gauges: Dict[str, Any] = {}
+        for resource in ('cpus', 'mem', 'disk', 'gpus'):
```

Comment:
For these and following, can we iterate over `ClustermanResources._fields`?  We do this in a few other spots and it'll clean up the code a bit and also make this more robust in the event that we add new resource types in the future.

Reply:
good call